parts/k8s/addons/kubernetesmasteraddons-calico-daemonset.yaml

```diff
@@ -190,323 +93,212 @@ spec:
     metadata:
       labels:
         k8s-app: calico-typha
-        addonmanager.kubernetes.io/mode: "EnsureExists"
```

Comment:
If you remove this the addon-manager won't create the object.

Reply:
From my test, I can see `calico-typha` pod running by just having label `addonmanager.kubernetes.io/mode: "EnsureExists"` set at `Deployment` metadata layer. This is similar to Deployment `calico-typha-horizontal-autoscaler` below.

So I am thinking omitting this label for `template.metadata` should be fine. WDYT?

Reply:
You're right 👍  I missed that it was not in the good section